configs/relengworker-prod/config.yml

```diff
@@ -117,3 +117,29 @@ worker_types:
         sla_seconds: 120
         capacity_ratio: 1.0
         min_replicas: 1
+
+  - worker_type: gecko-3-addon
+    provisioner: scriptworker-k8s
+    deployment_namespace: prod-addon
+    deployment_name: addon-prod-relengworker-firefox-1
+    autoscale:
+      algorithm: sla
+      args:
+        max_replicas: 10
+        avg_task_duration: 240
+        sla_seconds: 480
+        capacity_ratio: 1.0
+        min_replicas: 1
+
+  - worker_type: gecko-1-addon
+    provisioner: scriptworker-k8s
+    deployment_namespace: prod-addon
+    deployment_name: addon-prod-relengworker-fake-firefox-1 
+    autoscale:
+      algorithm: sla
+      args:
+        max_replicas: 10
+        avg_task_duration: 240
+        sla_seconds: 480
+        capacity_ratio: 1.0
+        min_replicas: 1
```

Comment:
I dug a bit more in some official docs and issues and turns out `min_replicas` defaults to 1 and can't be zero. There may be some additional tools to be installed within Kubernetes that could potentially enable that (via https://github.com/kubernetes/kubernetes/issues/69687 IIUC) but I'm not 100% sure. For now, I think we're safe keeping that to 1.

Reply:
Thanks for digging into that, agree that 1 is fine.